internal/cli/cli_test.go

```diff
@@ -101,3 +104,23 @@ func TestRun(t *testing.T) {
 		t.Errorf("cmd.Run was not executed")
 	}
 }
+
+func TestUsage(t *testing.T) {
+	c := &Command{
+		Short: "version prints the version information",
+		Usage: "librarian version",
+		Long:  "Command version prints version information for the librarian binary.",
```

Comment:
I got confused by the text here not matching the actual version command. (Possibly a lack of coffee...) Maybe use clearly-fake test command instead, e.g. "xyz tests the cli display" etc?